providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/steps/ProviderSteps.java

```diff
@@ -206,7 +206,7 @@ public void the_connection_is_lost_for(int seconds) throws InterruptedException,
         String randomizer = RandomStringUtils.randomAlphanumeric(5);
         String timoutName = "restart" + randomizer;
         Proxy proxy = toxiproxyClient.getProxy(generateProxyName(State.resolverType, state.providerType));
-        proxy.toxics().timeout(timoutName, ToxicDirection.UPSTREAM, seconds);
+        proxy.toxics().timeout(timoutName, ToxicDirection.DOWNSTREAM, seconds);
```

Comment:
I had to change this to downstream to fix my issue... hopefully it doesn't break the e2e tests in CI, but TBH this setting makes more sense to me. :thinking: